src/main/webapp/components/RoleManagementEdit.js

```diff
@@ -0,0 +1,151 @@
+import { Component, Fragment } from 'react';
+import { hh, h, button, div } from 'react-hyperscript-helpers';
+import { Modal, ModalHeader, ModalTitle, ModalFooter, ModalBody } from 'react-bootstrap';
+import { InputFieldCheckbox } from "./InputFieldCheckbox";
+import { ROLE, USER_ROLES } from '../util/roles';
```

Comment:
I'm digging through the code here to figure out what you're doing, and this is pretty confusing to me. You have two things (`USER_ROLES`, `ROLES`) that are really the same thing, roles that a user can have. Both are ordered the same, so I'm completely missing the point of the having the `ROLE` object at all. Probably outside the scope of this PR, but I think these two things should be merged into one data structure.

Reply:
I've removed `ROLE` constant object, and changed `USER_ROLES` in order to use only one data structure. You can check it [here](https://github.com/broadinstitute/orsp-pub/pull/178/files#diff-fda238a7164082c3e3529305f1f0ab37R1).